src/native/corehost/build.cmd

```diff
@@ -35,7 +36,8 @@ if /i [%1] == [amd64]       (set __BuildArch=x64&&shift&goto Arg_Loop)
 if /i [%1] == [arm64]       (set __BuildArch=arm64&&shift&goto Arg_Loop)
 
 if /i [%1] == [portable]    (set __PortableBuild=1&&shift&goto Arg_Loop)
-if /i [%1] == [rid]         (set __TargetRid=%2&&shift&&shift&goto Arg_Loop)
+if /i [%1] == [rid]         (set __OutputRid=%2&&shift&&shift&goto Arg_Loop)
+if /i [%1] == [outputrid]   (set __OutputRid=%2&&shift&&shift&goto Arg_Loop)
```

Comment:
Can you remove `rid` and  switch the build args passed on Windows to use `outputrid` instead?
https://github.com/dotnet/runtime/blob/14b844c06baabadcd0af7231015ee2321a51af59/src/native/corehost/corehost.proj#L136

Having both seems confusing and just keeping `outputrid` would match non-Windows better.